lms/djangoapps/student_account/views.py

```diff
@@ -504,10 +501,9 @@ def account_settings_context(request):
             # in with, or if the user is already authenticated with them.
         } for state in auth_states if state.provider.display_for_login or state.has_account]
 
-    try:
-        external_auth_map = ExternalAuthMap.objects.get(user=user)
-    except:
-        external_auth_map = None
-    context['is_shib_auth'] = 'shib' in external_auth_map.external_domain if external_auth_map else False
```

Comment:
@kluo Ah, so this is an issue of switching from `external_auth` to `third_party_auth`?

Reply:
Correct